yarn-project/noir-contracts/src/contracts/uniswap_contract/src/main.nr

```diff
@@ -59,12 +59,15 @@ contract Uniswap {
             l1UniswapPortal,
         ]);
 
-        // inputAsset.withdraw(inputAmount, l1UniswapPortal)
+        // inputAsset.withdraw(inputAmount, recipient=l1UniswapPortal, callerOnL1=l1UniswapPortal) // only uniswap portal can call this to safegaurd ordering of message consumption. 
+        // if someone else calls it on behalf of uniswap portal and the portal doesn't expect it, then funds might get locked! 
```

Comment:
Not sure I am following? If someone else calls it their transaction will simply revert, but that don't lock it. But yes, if the uniswap portal itself will not handle it, it could get locked, but think it is fine to remove this line and just keep the ref to docs.